composer.json

```diff
@@ -1,19 +1,23 @@
 {
-    "name": "helmutschneider/swish-php",
+    "name": "altitudeevents/swish-php",
     "description": "PHP-wrapper for the Swish api",
     "license": "MIT",
     "authors": [
         {
             "name": "Johan Björk",
             "email": "johanimon@gmail.com"
+        },
+        {
+            "name": "Tommie Lagerroos",
+            "email": "tommie@lagerroos.nu"
         }
     ],
     "minimum-stability": "stable",
     "require": {
         "php": ">=5.5.9",
         "ext-curl": "*",
         "ext-openssl": "*",
-        "guzzlehttp/guzzle": "~6.0"
+        "guzzlehttp/guzzle": "^7.2.0"
```

Comment:
Guzzle 7 probably requires a bump in the PHP version, right? 7.2.5 according to their docs.